### ISSUE TITLE
opensensemap sensor ids could be retrieved via API

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/opensesame#57]
`curl "https://api.opensensemap.org/boxes/64cb602193c69500072a580f?format=json" ` seems to return all sensor IDs, this could be executed at startup.

It is low-prior for now, as the first weather station should probably also work without this feature.

**Comment by fel115** [comment]
Yes, that's a better idea! So we only need to map the `sensorType` to the register addresses and it would be easier to use.
